lib/model/content.dart

```diff
@@ -297,7 +297,7 @@ class CodeBlockNode extends BlockContentNode {
   }
 }
 
-class CodeBlockSpanNode extends InlineContentNode {
+class CodeBlockSpanNode extends ContentNode {
```

Comment:
This should go in its own prep commit. It's rearranging this class hierarchy, which is a potentially more significant kind of change from replacing else-if with switch, so it should get its own commit message to highlight that change and the explanation of why it's good and OK.